cmd/cnab-to-oci/fixup.go

```diff
@@ -54,12 +47,12 @@ func runFixup(opts fixupOptions) error {
 	if err := json.Unmarshal(bundleJSON, &b); err != nil {
 		return err
 	}
-	resolver := createResolver(opts.insecureRegistries)
 	ref, err := reference.ParseNormalizedNamed(opts.targetRef)
 	if err != nil {
 		return err
 	}
-	if err := remotes.FixupBundle(context.Background(), &b, ref, resolver); err != nil {
+	err = remotes.FixupBundle(&b, ref, remotes.NewResolverConfigFromDockerConfigFile(config.LoadDefaultConfigFile(os.Stderr), opts.insecureRegistries...), remotes.WithEventCallback(displayEvent))
```

Comment:
nit: Would be nice to have this line a bit shorter. putting a `resolver := ...` on the previous line would help a lot.

_edit_: or even better adjust `createResolver` to fit instead of removing